SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -277,7 +277,7 @@ public void SelectAllocationsByWorkerEmail()
 
             _classUnderTest.CreateAllocation(createAllocationRequest);
```

Comment:
Would be good to have a test for fetching the allocations by `team_id` to ensure everything is working correctly

Reply:
Done in [9f3bf57](https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/610/commits/9f3bf57c4f2ed74f5302730c840c75520bb00a3a)